nomad/structs/structs.go

```diff
@@ -10599,18 +10582,6 @@ var MsgpackHandle = func() *codec.MsgpackHandle {
 	return h
 }()
 
-var (
```

Comment:
these had to move to another package because of the ordering of init() and module variables and the need to avoid circular imports.